### DISCUSSION TITLE
Volume command

### DISCUSSION THREAD

**Discussion by fabnavigator** [MusicPlayerDaemon/MPD#1698] [Q&A]
Why has the volume command been deprecated? I find it very useful for adjusting the volume.

**Comment by geneticdrift** [comment]
There's [`setvol`](https://mpd.readthedocs.io/en/latest/protocol.html#playback-options):
```
setvol {VOL}

    Sets volume to VOL, the range of volume is 0-100.
```

**Reply by fabnavigator** [reply]
My remote control has volume up and volume down buttons. setvol doesn't help with that. I would need to get the volume, increment or decrement it, and then set it. Lots of extra work for me.
